includes/class-wc-structured-data.php

```diff
@@ -269,6 +274,44 @@ public function generate_product_data( $product = null ) {
 				'ratingValue' => $product->get_average_rating(),
 				'reviewCount' => $product->get_review_count(),
 			);
+
+			// Markup most recent rating/review.
+			$comments = get_comments(
+				array(
+					'number'      => 1,
+					'post_id'     => $product->get_id(),
+					'status'      => 'approve',
+					'post_status' => 'publish',
+					'post_type'   => 'product',
+					'parent'      => 0,
+					'meta_key'    => 'rating',
+					'orderby'     => 'meta_value_num',
+				)
+			);
+
+			if ( $comments ) {
+				foreach ( $comments as $comment ) {
+					$rating = get_comment_meta( $comment->comment_ID, 'rating', true );
+
+					if ( ! $rating ) {
+						continue;
+					}
+
+					$markup['review'] = array(
+						'@type'        => 'Review',
+						'reviewRating' => array(
+							'@type'       => 'Rating',
+							'ratingValue' => $rating,
+							'bestRating'  => 5,
```

Comment:
bestRating and worstRating can actually be omitted since it defaults to 5 and 1 when omitted.